lib/resources/r4/immunization.yaml

```diff
@@ -414,3 +414,23 @@ fields:
     Will be set to true if not specified.
     For CREATE action should not be set to true when reportOrigin is populated.
     For UPDATE action should not be set to true.
+
+- name: protocolApplied.targetDisease
+  required: 'No'
+  type: CodeableConcept
+  url: https://www.hl7.org/fhir/r4/immunization-definitions.html#Immunization.protocolApplied.targetDisease
+  description: The vaccine preventable disease the dose is being administered against.
+  action: terminology
+  binding:
+    description: The vaccine preventable disease the dose is being administered against.
+    terminology:
+    - display: CVX
+      system: http://hl7.org/fhir/sid/cvx
+      info_link: http://hl7.org/fhir/r4/cvx.html
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+      info_link: http://hl7.org/fhir/r4/snomedct.html
```

Comment:
Do we actually have SNOMED mapping for this?

Reply:
Thanks for pointing it out. I think currently we do not have mappings but we do want to support it. @fpivonka can we leave it or should we add it once the mappings are completed?

Reply:
@lalitha94 @Mohammad-Isyroqi-Fathan I think we can leave it off for now. We do support the scope for SNOMED CT but we have not mapped clients that way.